utils/io/fileutils/files.go

```diff
@@ -133,6 +133,32 @@ func ListFilesRecursiveWalkIntoDirSymlink(path string, walkIntoDirSymlink bool)
 	return
 }
 
+// Return the list of files in the specified path that has the extension
+func ListFilesWithSpecificExtension(path, ext string) ([]string, error) {
+	sep := GetFileSeparator()
```

Comment:
I think that the *GetFileSeparator()* function shoukd be removed from thr code base and replaced with ```os.PathSeparator```

Reply:
Per our discussion, I'm keeping the function and just replacing with string(os.PathSeparator)